lib/Gocdb_Services/Config.php

```diff
@@ -94,88 +105,103 @@ public function setLocalInfoFileLocation($filePath){
          */
-        $this->local_info_xml = NULL;
+        $this->localInfoXml = null;
     }
 
     /**
      * Opens the gocdb_schema.xml file for reading, returning a simplexml object.
      * @return simple xml object
      */
-    public function GetSchemaXML() {
+    public function GetSchemaXML()
+    {
         return simplexml_load_file($this->getSchemaFileLocation());
     }
 
     /**
      * Returns a simplexml object representing the local_info.xml file
      * @return simple xml object
      */
-    private function GetLocalInfoXML() {
-        if ($this->local_info_xml == NULL) {
-            if (!($this->local_info_xml = $this->readLocalInfoXML($this->getLocalInfoFileLocation(), $this->local_info_override))) {
-                throw new \ErrorException("Failed to load xml configuration file: ".$this->getLocalInfoFileLocation());
+    private function GetLocalInfoXML()
+    {
+        if ($this->localInfoXml == null) {
+            $this->localInfoXml = $this->readLocalInfoXML($this->getLocalInfoFileLocation(), $this->localInfoOverride);
+            if (!$this->localInfoXml) {
+                throw new \ErrorException("Failed to load xml configuration file: " .
+                                            $this->getLocalInfoFileLocation());
             }
         }
-        return $this->local_info_xml;
+        return $this->localInfoXml;
     }
 
     /**
      * Reads the local_info.xml, returning a simplexml object. The contents of the file are
      * adjusted based on the url attribute provided.
      * @return simple xml object
      */
-    private function readLocalInfoXML ($path, $url = NULL) {
+    private function readLocalInfoXML($path, $url = null)
+    {
 
         libxml_use_internal_errors(true);
 
-        if (($base = simplexml_load_file($path)) == FALSE) {
-            $this->throwXmlErrors('Failed to load configuration file '.$path);
+        $base = simplexml_load_file($path);
+
+        if (!$base) {
+            $this->throwXmlErrors('Failed to load configuration file ' . $path);
         }
 
         // Search the input XML for a 'local_info' section that does NOT have a url attribute
         // specified. This is the default spec.
-        if (($unqualified = $base->xpath("//local_info[not(@url)]")) == FALSE) {
-            throw new \ErrorException('Failed to find local_info section without url in configuration file '.$path);
+        $unqualified = $base->xpath("//local_info[not(@url)]");
+        if (!$unqualified) {
+            throw new \ErrorException('Failed to find local_info section without url in configuration file ' . $path);
         }
 
         if (count($unqualified) != 1) {
-            throw new \ErrorException('Only one local_info element without url attribute is allowed in configuration file '.$path);
+            throw new \ErrorException(
+                'Only one local_info element without url attribute is allowed in configuration file ' .
```

Comment:
Given we are already concatinating here, can this long line be split up?

Reply:
It could be, but seems not overlong to me. Standard is set to max of 120 chars. What are you suggesting?

Reply:
it was long enough that it wrapped on GitHub for me 😢 

![image](https://user-images.githubusercontent.com/2488355/204591028-cd3584c3-b414-4cea-8991-7166345500a1.png)

Could go to <80? PSr12 does say lines SHOULD NOT be longer than that.

Reply:
i.e. 

```suggestion
                'Only one local_info element without url attribute is ' . 
                'allowed in configuration file ' .
```

Reply:
OK. I'm not a fan of 80 chars, being an arbitrary legacy, but I can see the point and it works here for concatenations.